### PR TITLE
First draft update of histogram documentation

### PR DESCRIPTION
**Pull request by lmoneta**

### CODE REVIEW

File: manual/histograms/index.md [hunk 1]
```diff
@@ -88,30 +87,31 @@ For all histogram types: `nbins`, `xlow`, `xup`:
...
 
 **Re-binning**
 
 You can re-bin a histogram via the [TH1::Rebin()](https://root.cern/doc/master/classTH1.html#aff6520fdae026334bf34fa1800946790){:target="_blank"} method. It returns a new histogram with the re-binned contents. If bin errors were stored, they are recomputed during the re-binning.
+You can see this [tutorial](https://root.cern/doc/master/rebin_8C.html) as a re-binning example.
 
 ### Stack of histograms
 
-  - {% include ref class="THStack" %} is a collection of {% include ref class="TH1" %} or {% include ref class="TH2" %} histograms.
+{% include ref class="THStack" %} is a collection of {% include ref class="TH1" %} or {% include ref class="TH2" %} histograms. The tutorial [hstack.C](https://root.cern/doc/master/hstack_8C.html) is a good example of how using the `THStack` class.
```
Comment (couet) [inline manual/histograms/index.md:107]: You can use: `{% include tutorial name="hstack" %}`to point to `hstack.C`

File: manual/histograms/index.md [hunk 2]
```diff
@@ -197,84 +268,114 @@ Creating a new histogram without changing the original one:
...
-- Use the [GetBinWidth()](https://root.cern/doc/master/classTH1.html#ad69a5fa0002361fd77f37990a29d1aa3){:target="_blank"} method to get the bin width of a histogram.
+To use the normalization methods, you can clone first the histogram to keep the original one, call then `TH1::Scale` passing as scale parameter value the histogram integral. In addition, use the option `width` to divide also by the bin width in order to display the probability density in each bin.
+If you want to show just the frequency probability of each bin, you don't need to use the `width` option.
 
 {% highlight C++ %}
-   TH1F h1("h1","Histogram from a Gaussian",100,-3,3);
-   h1.FillRandom("gaus",10000);
-   h1.Draw();
-   h1.GetBinWidth(0)
+TH1*h1 = (TH1*)(h->Clone("h1"));
```
Comment (couet) [inline manual/histograms/index.md:303]: usually we ended a bit the code(3 spaces)

File: manual/histograms/index.md [hunk 3]
```diff
@@ -197,84 +268,114 @@ Creating a new histogram without changing the original one:
...
+- Use the [TH1::Draw()](https://root.cern/doc/master/classTH1.html#aa53a024a9e94d5ec91e3ef49e49563da){:target="_blank"} method to draw a histogram using the provided drawing option.
+    The drawing is delegated to the {% include ref class="THistPainter" %} class that specializes the drawing of the histogram. The {% include ref class="THistPainter" %} class is separated
+    from the histogram, so that the histogram class does not contain the graphics overhead.
+
+### Drawing options
 
 The "drawing option" is the unique parameter of the [TH1::Draw()](https://root.cern/doc/master/classTH1.html#aa53a024a9e94d5ec91e3ef49e49563da){:target="_blank"}
 method. It specifies how the histogram will be graphically rendered.
 For detailed information on the drawing options for all histogram classes, refer to
 [THistPainter](https://root.cern/doc/master/classTHistPainter.html#HP01a){:target="_blank"}.
```
Comment (couet) [inline manual/histograms/index.md:328]: You can use: `{% include ref class="THistPainter" %}`

File: manual/histograms/index.md [hunk 4]
```diff
@@ -197,84 +268,114 @@ Creating a new histogram without changing the original one:
...
+{% highlight C++ %}
+{
+   TH2D h2("h2","Histogram filled with random numbers",40,-4,4,40,-20,20);
    float px, py;
    for (int i = 0; i < 25000; i++) {
       gRandom->Rannor(px,py);
       h2.Fill(px,5*py);
    }
-   h2.Draw("LEGO1");
+   h2.DrawCopy("LEGO1");
```
Comment (couet) [inline manual/histograms/index.md:371]: Why not using pointers ? that will avoid the use of `DrawCopy` ?

**Review by couet — APPROVED** [review]

File: manual/histograms/index.md (hunk 4 above)
Comment (lmoneta) [reply manual/histograms/index.md:371]: I added this as an example when DrawCopy is needed

File: manual/histograms/index.md (hunk 1 above)
Comment (lmoneta) [reply manual/histograms/index.md:107]: Thank you , I will update the links

File: manual/histograms/index.md (hunk 3 above)
Comment (lmoneta) [reply manual/histograms/index.md:328]: Here I wanted to reference specific paragraph in the THistPainter class

**Pull request merged**